### PR TITLE
Contextembeds

### PR DESCRIPTION
**Pull request by bplank**
update code to support `--embeds_in_file` as additional (contextualized) word embeddings

### CODE REVIEW

File: src/structbilty.py [hunk 1]
```diff
@@ -640,23 +659,36 @@ def get_features(self, seqs, train=False, update=True, embeds_in_file=False):
...
             cfeatures = [self.get_c_repr(word, train=train) for word in words]
+
             if len(lex_features) > 0:
                 lex_features = dynet.inputTensor(lex_features)
                 features = [dynet.concatenate([w,c,l]) for w,c,l in zip(wfeatures,cfeatures,lex_features)]
             else:
                 features = [dynet.concatenate([w, c]) for w, c in zip(wfeatures, cfeatures)]
+            if embeds_in_file:
+                if len(lex_features) > 0:
+                    print("both contextualized embeddings and lexical features are not yet supported")
```
Comment (robvanderg) [inline src/structbilty.py]: How about: "Using both contextualized embeddings and lexical features simultaneously is not yet supported"?
This is ambiguous ;)

Comment (bplank) [reply src/structbilty.py]: good point, this is no longer the case, we can drop this.

Comment (bplank) [reply src/structbilty.py]: removed this. 

Now --embeds_in_file_dim is only needed during training.

**Comment by bplank** [comment]
merging `contextembeds` into main branch

**Pull request merged**